src/plugins/telemetry/server/telemetry_collection/get_cluster_stats.test.ts

```diff
@@ -9,20 +9,23 @@
 
 import { elasticsearchServiceMock } from '@kbn/core/server/mocks';
 import { getClusterStats } from './get_cluster_stats';
-import { TIMEOUT } from './constants';
+import { CLUSTER_STAT_TIMEOUT } from './constants';
 
 describe('get_cluster_stats', () => {
   it('uses the esClient to get the response from the `cluster.stats` API', async () => {
     const response = { cluster_uuid: '1234' };
     const esClient = elasticsearchServiceMock.createClusterClient().asInternalUser;
     esClient.cluster.stats.mockImplementationOnce(
       // @ts-expect-error the method only cares about the response body
-      async (_params = { timeout: TIMEOUT }) => {
+      async (_params = { timeout: CLUSTER_STAT_TIMEOUT }) => {
```

Comment:
TBH, I'm not sure why we need to default the params' `timeout` here... but that's not on you 😅

Reply:
thanks @afharo , good point. @quux00 could this `timeout` param be omitted altogether?

Reply:
I mean in this test mock. Not the actual production code. Sorry if I was not clear.